### PR TITLE
fix(users): user feedback messages improvements

### PR DESCRIPTION
**Pull request by curzolapierre**
~~- [ ] Add a changelog entry in the section "To Be Released" of CHANGELOG.md~~

### CODE REVIEW

File: db/users/update.go [hunk 1]
```diff
@@ -47,6 +47,17 @@ func UpdateUser(ctx context.Context, app, addonUUID, username string) error {
 	if !userExists {
 		return errors.New(ctx, fmt.Sprintf("User \"%s\" does not exist", username))
 	}
+	// Check if the user is protected
```
Comment (ipfaze) [inline db/users/update.go:50]: issue(blocking): you can avoid a loop here by saving the user in the last for loop (https://github.com/Scalingo/cli/blob/2b8d41467bf986a62428527f4232f6cd5fa4c36e/db/users/update.go#L41).
And then do you check after that.

**Review by ipfaze — CHANGES_REQUESTED** [review]

File: db/users/update.go [hunk 2]
```diff
@@ -36,18 +36,27 @@ func UpdateUser(ctx context.Context, app, addonUUID, username string) error {
...
 		}
 	}
 	if !userExists {
 		return errors.New(ctx, fmt.Sprintf("User \"%s\" does not exist", username))
 	}
 
+	if userProtected {
+		return errors.New(ctx, fmt.Sprintf("User \"%s\" is protected and cannot be updated", username))
+	}
+
```
Comment (ipfaze) [inline db/users/update.go:59]: For better readability I suggest:
```suggestion
    var user
    for _, u := range l {
	    if u.Name == username {
		    user = u
		    break
	    }
    }
    if user == nil {
	    return errors.New(ctx, fmt.Sprintf("User \"%s\" does not exist", username))
    }
    if user.Protected {
	    return errors.New(ctx, fmt.Sprintf("User \"%s\" is protected", username))
    }
```
WDYT?

Comment (curzolapierre) [reply db/users/update.go]: I pushed a suggestion, WDYT?

**Review by ipfaze — APPROVED** [review]
LGTM :+1:

**Pull request merged**